app/controllers/cookie_preferences_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class CookiePreferencesController < ApplicationController
```

Comment:
Still not sure about the name `CookiePreferences`. It certainly makes sense to me, but are we happy with the path name `/cookie-preferences/edit`? Should it be just `/cookies` using `CookiesController#show`?

Reply:
In what context would this url appear? we already have a cookies page fwiw